src/Raven.Server/Documents/Handlers/AI/Agents/ConversationDocument.cs

```diff
@@ -18,27 +19,60 @@ public class ConversationDocument(string agent, BlittableJsonReaderObject parame
     
     public BlittableJsonReaderObject Parameters = parameters;
     public List<BlittableJsonReaderObject> Messages = [];
-    public List<string> HistoryDocuments = [];
+    public List<string> LinkedConversations = [];
     public Dictionary<string, AiAgentActionRequest> OpenActionCalls = [];
     public AiUsage TotalUsage = new AiUsage();
     public string ChangeVector;
     public string Id;
-    public void Initialize(JsonOperationContext context, AiAgentConfiguration configuration, string userPrompt)
+
+    public DateTime LastMessageAt;
+    public DateTime CreatedAt = DateTime.UtcNow;
+    public void Initialize(JsonOperationContext context, AiAgentConfiguration configuration)
     {
         if (Messages.Count > 0)
             throw new InvalidOperationException("conversation document is already initialized. Cannot re-initialize.");
 
         foreach (var parameter in configuration.Parameters)
         {
             if (Parameters == null || Parameters.TryGet(parameter.Name, out object _) == false)
-                throw new ArgumentException($"Parameter '{parameter}' is missing.");
+                throw new ArgumentException($"Parameter '{parameter.Name}' is missing.");
         }
 
         AddMessage(context, context.ReadObject(new DynamicJsonValue
         {
-            ["role"] = "system",
-            ["content"] = configuration.SystemPrompt
+            [ChatCompletionClient.Constants.RequestFields.Role] = ChatCompletionClient.Constants.RequestFields.RoleSystemValue,
+            [ChatCompletionClient.Constants.RequestFields.Content] = configuration.SystemPrompt
         }, "system/msg"), usage: null);
+
+        if (TryCreateParameterDescriptionMessage(configuration, out string message))
+        {
+            AddMessage(context, context.ReadObject(new DynamicJsonValue
+            {
+                [ChatCompletionClient.Constants.RequestFields.Role] = ChatCompletionClient.Constants.RequestFields.RoleSystemValue,
+                [ChatCompletionClient.Constants.RequestFields.Content] = message
+            }, "system/msg"), usage: null);
+        }
+
+        if (configuration.Parameters.Count > 0)
+        {
+            AddMessage(context, context.ReadObject(new DynamicJsonValue
+            {
+                [ChatCompletionClient.Constants.RequestFields.Role] = ChatCompletionClient.Constants.RequestFields.RoleUserValue,
+                [ChatCompletionClient.Constants.RequestFields.Content] = ParametersToString(configuration)
+            }, "system/msg"), usage: null);
+        }
+    }
+
+    private string ParametersToString(AiAgentConfiguration configuration)
+    {
+        var sb = new StringBuilder();
+        foreach (var parameter in configuration.Parameters)
+        {
+           var value = Parameters[parameter.Name];
+           sb.AppendLine($"the value of the '{parameter.Name}' parameter is: {value.ToString()}");
```

Comment:
No need for the extra tokens

```suggestion
           sb.AppendLine($"{parameter.Name} = {value.ToString()}");
```